src/devices/src/virtio/net/device.rs

```diff
@@ -652,7 +639,6 @@ impl VirtioDevice for Net {
     fn read_config(&self, offset: u64, mut data: &mut [u8]) {
         let config_len = self.config_space.len() as u64;
         if offset >= config_len {
-            error!("Failed to read config space");
```

Comment:
Reading and writing the config space should not be on the critical path. We should keep this log and the next one.